linearize-0.1.3 seems to not build with arbitrary-1 feature enabled and arbitrary 1.3

I'm pulling in linearize 0.1.3 with arbitrary-1 feature enabled and it's failing to compile


```
error[E0432]: unresolved import `arbitrary_1::MaxRecursionReached`
 --> /home/shana/.cargo/registry/src/index.crates.io-6f17d22bba15001f/linearize-0.1.3/src/foreign/arbitrary_1.rs:3:41
  |
3 |     arbitrary_1::{size_hint, Arbitrary, MaxRecursionReached, Unstructured},
  |                                         ^^^^^^^^^^^^^^^^^^^ no `MaxRecursionReached` in the root
```

--- a/linearize/Cargo.toml
+++ b/linearize/Cargo.toml
@@ -25,7 +25,7 @@ rand-0_8 = ["dep:rand-0_8"]
 cfg-if = "1"
 linearize-derive = { path = "../linearize-derive", version = "0.1", optional = true }
 serde-1 = { package = "serde", version = "1", default-features = false, optional = true }
-arbitrary-1 = { package = "arbitrary", version = "1", default-features = false, optional = true }
+arbitrary-1 = { package = "arbitrary", version = "1.4", default-features = false, optional = true }
 bytemuck-1 = { package = "bytemuck", version = "1", default-features = false, optional = true }
 rand-0_8 = { package = "rand", version = "0.8", default-features = false, optional = true }
 

--- a/linearize/Cargo.toml
+++ b/linearize/Cargo.toml
@@ -26,7 +26,7 @@ cfg-if = "1"
 linearize-derive = { path = "../linearize-derive", version = "0.1", optional = true }
 serde-1 = { package = "serde", version = "1", default-features = false, optional = true }
 arbitrary-1 = { package = "arbitrary", version = "1.4", default-features = false, optional = true }
-bytemuck-1 = { package = "bytemuck", version = "1", default-features = false, optional = true }
+bytemuck-1 = { package = "bytemuck", version = "1.9", default-features = false, optional = true }
 rand-0_8 = { package = "rand", version = "0.8", default-features = false, optional = true }
 
 [build-dependencies]

--- a/linearize/Cargo.toml
+++ b/linearize/Cargo.toml
@@ -27,7 +27,7 @@ linearize-derive = { path = "../linearize-derive", version = "0.1", optional = t
 serde-1 = { package = "serde", version = "1", default-features = false, optional = true }
 arbitrary-1 = { package = "arbitrary", version = "1.4", default-features = false, optional = true }
 bytemuck-1 = { package = "bytemuck", version = "1.9", default-features = false, optional = true }
-rand-0_8 = { package = "rand", version = "0.8", default-features = false, optional = true }
+rand-0_8 = { package = "rand", version = "0.8.3", default-features = false, optional = true }
 
 [build-dependencies]
 version_check = "0.9.5"

